lldb/docs/resources/lldbgdbremote.md

```diff
@@ -1998,6 +1998,19 @@ threads (live system debug) / cores (JTAG) in your program have
 stopped and allows LLDB to display and control your program
 correctly.
 
+## qWasmCallStack
+
+Get the Wasm callback for the given thread id. This returns a hex-encoding list
+of 64-bit addresses for the frame PCs. To match the Wasm specification, the
+addresses are encoded in little endian byte order.
+
```

Comment:
Also add a note here about the origin of this packet. Given that we're adding it to our extensions but if, say, GDB wanted to add this, we would in fact tell them to go look at `<whatever runtime originated this>`.

(it done makes some to be in this doc because it's not part of GDB's remote protocol spec)